packages/tracking/src/hooks/use_track_page_view.ts

```diff
@@ -9,9 +9,9 @@ export const useTrackPageView = () => {
 
   useEffect(() => {
     const match = pageMapping.find((m) => m.selector.test(location.pathname));
-    if (match?.track) {
-      track('gav4_pageView', {
-        ...match.track,
+    if (match?.page_track) {
+      track('page_view', {
+        page_name: match.page_name,
```

Comment:
this will later on be injected in track anyways meanwhile.. but probably good to have it here explicitly... otherwise just a simple track without payload?

Reply:
setting it here explicit since `page_view` expects a `page_name` in the tracking payload (allowing empty events would be confusing in this case)